### ISSUE TITLE
Cannot set `bind_host` in AdGuardHome.yaml (docker version)

### ISSUE THREAD

**Issue by pexcn** [AdguardTeam/AdGuardHome#4231]
### Issue Details

* **Version of AdGuard Home server:**
  * v0.107.3 (docker)
* **How did you install AdGuard Home:**
  * docker
* **How did you setup DNS configuration:**
  * Here is my config file: https://github.com/pexcn/docker-images/blob/master/ext/adguardhome/adguardhome-data/conf/AdGuardHome.yaml
  * Use docker-compose, here is my compose config: https://github.com/pexcn/docker-images/blob/master/ext/adguardhome/docker-compose.yml
* **CPU architecture:**
  * AMD64
* **Operating system and version:**
  * OpenWrt

Hi guys:
Now I am using v0.107.3 version, I want to change `bind_host` to `192.168.1.1`, but after restarting adguardhome, the value in the configuration file will automatically change to `0.0.0.0`.

Maybe it has something to do with this line of the Dockerfile https://github.com/AdguardTeam/AdGuardHome/blob/v0.107.3/scripts/make/Dockerfile#L59, can it be fixed in the future? :smile:

**Comment by pexcn** [comment]
Is there an update on this issue? 😀 @ainar-g

**Comment by jonathanjsimon** [comment]
Necrobumping

With the -h 0.0.0.0 in the Dockerfile, AdGuard becomes unusable in my use case as I have two interfaces and port 80 is in use on one of them. Specifying the other interface's IP is the only way to use AdGuard and have individual client IPs listed in the interface.

**Comment by jonathanjsimon** [comment]
For anyone wanting to do this now, https://hub.docker.com/r/jsimon0/adguardhome

**Comment by MarioNoll** [comment]
Docker also allows [overriding](https://docs.docker.com/engine/reference/run/#overriding-dockerfile-image-defaults) image defaults. So as a workaround I am using a custom command in my compose file omitting the host option:

```
command: [  "--no-check-update",  "-c",  "/opt/adguardhome/conf/AdGuardHome.yaml", "-w",  "/opt/adguardhome/work" ]
```

**Comment by Mizzick** [comment]
Please have a look, we have implemented the requested changes.
The new build version `v0.108.0-a.579+e54fc9b1` has been just published to the edge channel.

**Comment by ainar-g** [comment]
This should be fixed as of v0.107.34. Can any of you please check?

**Comment by MarioNoll** [comment]
@ainar-g Works fine in v.0.107.33, thanks!
